src/app/routes/utils/fetchPageData/utils/getRouteProps/index.js

```diff
@@ -34,12 +34,17 @@ const getRouteProps = url => {
   const variant = variantSanitiser(variantPath);
   const errorCode = path(['errorCode'], params);
 
+  const cpsId =
+    service && assetUri
+      ? `${service}${variant ? `/${variant}` : ''}/${assetUri}`
+      : undefined;
+
   return {
     isAmp: amp || isAmpPath(url),
     isApp: app || isAppPath(url),
     service: service || fallbackServiceParam(url),
     variant,
-    id,
+    id: id || cpsId,
     assetUri,
```

Comment:
Would like to combine these 2 into one value, maybe called `assetId`.

`id` appears to only be used in the `ArticleMediaPlayer` component: https://github.com/bbc/simorgh/blob/latest/src/app/legacy/containers/ArticleMediaPlayer/index.jsx#L11 

`assetUri` doesn't appear to be used or passed down to our Contexts either: https://github.com/bbc/simorgh/blob/latest/src/app/legacy/containers/PageHandlers/withContexts.jsx

Reply:
Appears to be straightforward to combine them, just requires updating quite a few references and tests that set the `id` field manually.